.github/workflows/build_release.yml

```diff
@@ -0,0 +1,133 @@
+            cat Cargo.toml
+          fi
+          if [[ "${{ matrix.runner }}" == "ubuntu-latest" ]]; then
+            sed -i '/\[package\]/,/^version = /s/^version = .*/version = \"${{ github.event.release.tag_name }}\"/' Cargo.toml
+            cat Cargo.toml
+          fi
+
+      - name: Prepare for ARM64
+        if: ${{ matrix.target == 'aarch64-unknown-linux-gnu' }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc-aarch64-linux-gnu
+          
+          # Define the path to the Cargo config file
+          CARGO_CONFIG="$HOME/.cargo/config.toml"
+
+          # Check if the file exists; if not, create it
+          if [[ ! -f "$CARGO_CONFIG" ]]; then
+              mkdir -p "$HOME/.cargo"
+              touch "$CARGO_CONFIG"
+          fi
+
+          # Add the target configuration if it doesn't already exist
+          if ! grep -q "\[target.aarch64-unknown-linux-gnu\]" "$CARGO_CONFIG"; then
+              echo -e "\n[target.aarch64-unknown-linux-gnu]" >> "$CARGO_CONFIG"
+              echo 'linker = "aarch64-linux-gnu-gcc"' >> "$CARGO_CONFIG"
+              echo "Configuration added to $CARGO_CONFIG"
+          else
+              echo "Configuration already exists in $CARGO_CONFIG"
+          fi
+          cat $HOME/.cargo/config.toml
+
+      - name: Build Binary
+        run: cargo build --verbose --release --target ${{ matrix.target }} # --locked have been removed to avoid error
+      
+      - name: Dry run cargo publish
+        run: cargo publish --dry-run --allow-dirty --target ${{ matrix.target }}
+      
+      - name: Upload Binaries
+        uses: "taiki-e/upload-rust-binary-action@v1"
+        with:
+          bin: ${{ env.PROJECT_NAME }}
+          target: ${{ matrix.target }}
+          token: ${{ secrets.TOKEN_GITHUB }}
+
+      - name: Publish package to crates.io
+        run: cargo publish --token ${CRATES_TOKEN} --target ${{ matrix.target }}
+        env:
+          CRATES_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+
+  docker-build-push:
+    runs-on: ubuntu-latest
+    timeout-minutes: 70
+    needs: build-cross-package
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      
+      - name: Build & Push cedar-agent
+        uses: docker/build-push-action@v4
+        with:
+          file: Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: true
+          cache-from: type=registry,ref=permitio/cedar-agent:latest
+          cache-to: type=inline
+          tags: |
+            permitio/cedar-agent:latest
```

Comment:
Shouldn't this only be if it's a latest release ?

Reply:
It's have latest release and the version. where is the problem with that?